### PR TITLE
Complete `circle-*` set

### PR DESCRIPTION
**Pull request by danielbayley**
`circle-equal` can double as `debug-breakpoint-*` from [codicons](https://code.visualstudio.com/api/references/icons-in-labels#icon-listing)

### CODE REVIEW

File: icons/circle-equal.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <line x1="8" x2="16" y1="9" y2="9" />
+  <line x1="8" x2="16" y1="15" y2="15" />
```
Comment (karsa-mistmere) [inline icons/circle-equal.svg:13]: I think this has to be about the same ratio as our regular `equals` icon, so maybe something more like this?
```suggestion
<path d="M7 10H17" />
<path d="M7 14H17" />
```

File: icons/circle-slash.svg
Comment (karsa-mistmere) [inline icons/circle-slash.svg:1]: A bit unsure about the nomenclature of this one, we already have a `circle-slashed` and a `slash` (that's actually also a `circle-slash`). Maybe `circle-slash-inside`?

Comment (karsa-mistmere) [reply icons/circle-slash.svg:1]: Naaah, `circle-off` would be totally different, see below. :D
image

`slash` for sure is a misnomer, but [I don't like `no-entry` either](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRfL8Md7FrU1s8IzEqtPGPH5M2LaJnTF44NDg&usqp=CAU).
Maybe `ban`?

Comment (danielbayley) [reply icons/circle-slash.svg:1]: > `circle-off` would be totally different

Yeah I also meant it’s probably styled wrong too… or do we need both?

> Maybe `ban`?

Works for me…

**Review by karsa-mistmere — APPROVED** [review]
I just brought up `circle-off` as an example, but I guess it doesn't hurt, I'm OK with merging this as is as long as @ericfennis is.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**